backend/api/views/annotation_task.py

```diff
@@ -48,37 +43,58 @@ class AnnotationTaskViewSet(viewsets.ViewSet):
+                                                d.name as dataset_name,
+                                                a.start,
+                                                a."end"
+                                         FROM dataset_files file
+                                         LEFT JOIN datasets d on file.dataset_id = d.id
+                                         LEFT JOIN audio_metadata a on file.audio_metadatum_id = a.id
+                                        ) file on file.id = task.dataset_file_id
+                                      WHERE annotator_id=%s AND annotation_campaign_id=%s
+                                      ORDER BY file.start""",
+            (request.user.id, campaign_id),
+        )
 
-    @extend_schema(
-        parameters=[
-            OpenApiParameter("campaign_id", int, OpenApiParameter.PATH),
-            OpenApiParameter("user_id", int, OpenApiParameter.PATH),
-        ],
-        responses=AnnotationTaskSerializer(many=True),
-    )
-    @action(
-        detail=False,
-        url_path="campaign/(?P<campaign_id>[^/.]+)/user/(?P<user_id>[^/.]+)",
-    )
-    def campaign_user_list(self, request, campaign_id, user_id):
-        """List tasks for given annotation campaign and user"""
-        annotation_campaign = get_object_or_404(AnnotationCampaign, pk=campaign_id)
-        _user = get_object_or_404(User, pk=user_id)
-        if not request.user.is_staff and not request.user == annotation_campaign.owner:
-            return HttpResponse("Unauthorized", status=403)
-
-        queryset = self.queryset.filter(
-            annotator_id=user_id, annotation_campaign_id=campaign_id
-        ).prefetch_related("dataset_file", "dataset_file__dataset")
         serializer = self.serializer_class(queryset, many=True)
         return Response(serializer.data)
 
+    # @extend_schema(
+    #     parameters=[
+    #         OpenApiParameter("campaign_id", int, OpenApiParameter.PATH),
+    #         OpenApiParameter("user_id", int, OpenApiParameter.PATH),
+    #     ],
+    #     responses=AnnotationTaskSerializer(many=True),
+    # )
+    # @action(
+    #     detail=False,
+    #     url_path="campaign/(?P<campaign_id>[^/.]+)/user/(?P<user_id>[^/.]+)",
+    # )
+    # def campaign_user_list(self, request, campaign_id, user_id):
+    #     """List tasks for given annotation campaign and user"""
+    #     annotation_campaign = get_object_or_404(AnnotationCampaign, pk=campaign_id)
+    #     _user = get_object_or_404(User, pk=user_id)
+    #     if not request.user.is_staff and not request.user == annotation_campaign.owner:
+    #         return HttpResponse("Unauthorized", status=403)
+    #
+    #     queryset = self.queryset.filter(
+    #         annotator_id=user_id, annotation_campaign_id=campaign_id
+    #     ).prefetch_related("dataset_file", "dataset_file__dataset")
+    #     serializer = self.serializer_class(queryset, many=True)
+    #     return Response(serializer.data)
```

Comment:
To be honest, I didn't removed this code because I first didn't know what was its purpose. But i'll clean it.
It would be great to fix the swagger! Not the priority though, I made a ticket for that.